frameworks/mlr3automl/setup.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 HERE=$(dirname "$0")
 VERSION=${1:-"stable"}
-REPO=${2:-"a-hanf/mlr3automl"}
+REPO=${2:-"ja-thomas/mlr3automl"}
```

Comment:
no need to change the default repo, I'd prefer to leave the official repo as the default.
On our side, we can easily change this directly in the framework definition:
```yaml
mlr3automl:
  setup_args: ["master", "ja-thomas/mlr3automl"]  # or use "#commit-hash" instead of "master"
  version: 'ja-thomas-fix'
```
@PGijsbers wdyt?

Reply:
I agree. I prefer to give @a-hanf a few days to see if Janek's fixes can be merged with the main mlr3automl repo, but otherwise we will tackle this through the framework definition.

Reply:
Hi everyone and thanks for looking into this.
I merged Janek's changes to the mlr3automl main branch and tried a local test run of the benchmark. The set-up went through without errors and the code was running, but the results show some errors.
I'll look into this more in the following days, hopefully we can get it resolved soon.
Thanks!